### PR TITLE
add emr yarn cluster for spark gpu

### PR DESCRIPTION
**Pull request by lanking520**
Add yarn cluster configuration for EMR on GPU

### CODE REVIEW

File: aws/emr-distributed-inference/image-classification-gpu/src/main/scala/com/examples/ImageClassificationExample.scala [hunk 1]
```diff
@@ -92,6 +94,12 @@ object ImageClassificationExample {
 
     // download images
     val imagePath = downloadImages(Files.createTempDirectory("images"))
```
Comment (frankfliu) [inline aws/emr-distributed-inference/image-classification-gpu/src/main/scala/com/examples/ImageClassificationExample.scala:96]:
```suggestion
    val destPath = new org.apache.hadoop.fs.Path("hdfs:///images")
    downloadImages(destPath);
```

File: aws/emr-distributed-inference/image-classification-gpu/src/main/scala/com/examples/ImageClassificationExample.scala [hunk 2]
```diff
@@ -92,6 +94,12 @@ object ImageClassificationExample {
...
+    // upload to hadoop
```
Comment (frankfliu) [inline aws/emr-distributed-inference/image-classification-gpu/src/main/scala/com/examples/ImageClassificationExample.scala:97]: Move this section into downloadImages()

File: aws/emr-distributed-inference/image-classification-gpu/build.gradle [hunk 3]
```diff
@@ -12,10 +10,18 @@ repositories {
...
+configurations {
+    exclusion
+}
+
 dependencies {
     implementation platform("ai.djl:bom:0.8.0")
     implementation "org.apache.spark:spark-sql_2.12:3.0.0"
     implementation "org.apache.spark:spark-mllib_2.12:3.0.0"
+    implementation "org.apache.hadoop:hadoop-hdfs:2.7.4"
+    exclusion "org.apache.spark:spark-sql_2.12:3.0.0"
```
Comment (stu1130) [inline aws/emr-distributed-inference/image-classification-gpu/build.gradle:22]: could you add a comment why we need to exclude these dependencies?

File: aws/emr-distributed-inference/image-classification-gpu/src/main/scala/com/examples/ImageClassificationExample.scala [hunk 4]
```diff
@@ -81,17 +83,25 @@ object ImageClassificationExample {
     override def getBatchifier: Batchifier = Batchifier.STACK
   }
 
-  def downloadImages(outputPath : Path) : String = {
+  def downloadImages(outputPath :  Path) : String = {
```
Comment (stu1130) [inline aws/emr-distributed-inference/image-classification-gpu/src/main/scala/com/examples/ImageClassificationExample.scala:86]: nit: remove extra space

**Review by stu1130 — APPROVED** [review]

**Pull request merged**